flask_monitoringdashboard/core/config/__init__.py

```diff
@@ -27,6 +27,7 @@ def __init__(self):
         """Sets the default values for the project."""
         # dashboard
         self.version = '1.0'
+        self.blueprint_name = os.environ.get('FMD_BLUEPRINT', 'dashboard')
```

Comment:
sorry, why don't we read this from the config file?

Reply:
I agree with mircea that this should come from the config file

Reply:
The `init_from` method of the `Config` class is called by the user after the FMD is imported. However, the blueprint is created in FMD's `__init__` itself, so we need the blueprint name there. If we want to read it from a file at import time we need to either: 
a) fix the location of the config file itself 
b) fix the name of the env var to point to the location of the config file.
Now I'm doing option b)  but minus the extra step of pointing to a file, I'm giving the value directly.

Reply:
ouch! i see now.

Reply:
wouldn't it work if the blueprint would be initialized with None 
and then within the `.bind` that global blueprint would be overwritten with whatever's coming from config if there's something ?

Reply:
Good idea, it should work. I'll give it a try

Reply:
@mircealungu Turns out it's not a good idea to create a new blueprint object with every call of `bind`.  The first call of `bind` works fine, but then starting with the second call, the url's don't get registered anymore, hence the failing tests. Scratched my head for a while as to why, but in the end it was enough to just keep the global blueprint and only change its name in `bind`.

Reply:
Changing the name sounds much more elegant! Nice :)